app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,70 @@
+
+<main>
+    <section class="spotlight">
+  <h2 class="spotlight__header">
+    <span class="spotlight__header--prefix">Media Spotlight: </span>
+    <%= link_to @spotlight.title, work_path(@spotlight.id), class:"spotlight__link-to" %> by <%= @spotlight.creator %>
+  </h2>
+  <p class="spotlight__description">
+    <%= @spotlight.votes.length %> votes - <%= @spotlight.description %>
+  </p>
+</section>
+
+<hr class="root__hr" />
+
+<section class="top-ten__container">
+<section class="top-ten__list-container">
+<h2 class="top-ten__header">
+  Top Movies
+</h2>
+<% @movies.each do |movie|  %>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?